### DISCUSSION TITLE
My student application has been approved for over 72 hours, but I still haven't received any benefits

### DISCUSSION THREAD

**Discussion by liangjiedouble** [community/community#138764] [GitHub Education]
### Hi everyone,

The free GitHub Education benefit I applied for on September 13th has been approved for over 72 hours, but I still cannot use the benefit. Hope to receive assistance.

**Comment by davevad93** [comment]
Hi @liangjiedouble ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @liangjiedouble was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by liangjiedouble** [reply]
That's OK. Thank you very much!
